web/plugs/analytics_identify.ex

```diff
@@ -5,12 +5,11 @@ defmodule CodeCorps.Plug.AnalyticsIdentify do
 
   def init(opts), do: opts
 
-  def call(conn, _opts) do
-    if current_user = conn.assigns[:current_user] do
-      CodeCorps.Analytics.Segment.identify(current_user)
-      conn
-    else
-      conn
-    end
+  def call(conn, _opts), do: conn |> identify
+
+  def identify(%{assigns: %{current_user: user}} = conn) do
```

Comment:
@sbatson5 I guess there's no reason why it can't be. Can you adjust your review to needs changes then so I know to change it?